### PR TITLE
Progress page action bar

### PR DESCRIPTION
**Pull request by maurizi**

### CODE REVIEW

File: src/nyc_trees/apps/survey/urls/blockface.py [hunk 1]
```diff
@@ -31,6 +34,11 @@
     url(r'^(?P<blockface_id>\d+)/reservation-popup/$',
         reserved_blockface_popup, name='reserved_blockface_popup'),
 
+    # Note: changes here must be kept in sync with
+    # src/nyc_trees/js/src/reservationPage.js
```
Comment (maurizi) [inline src/nyc_trees/apps/survey/urls/blockface.py]: Note to self, wrong JS file

**Comment by jwalgran** [comment]
The "reserve it" link does not have the correct href

screen shot 2015-02-17 at 2 00 34 pm

**Comment by maurizi** [comment]
@jwalgran what should it link to?  @RickMohr just had it as `#`

**Comment by jwalgran** [comment]
You can zoom to it on the reservation page by using the URL hash http://localhost:8000/blockface/#24333

**Comment by jwalgran** [comment]
+1

**Pull request merged**